### PR TITLE
dev/core#488 Add in property for addresses, describing the level of d…

### PR DESCRIPTION
**Pull request by seamuslee001**
…isability access that is avaliable

Overview
----------------------------------------
This adds in a new column to the civicrm_address table to be able to store information about the level of disability access an address has

Before
----------------------------------------
No information on the level of access

After
----------------------------------------
Level of disability access is able to be stored

ping @eileenmcnaughton @agileware @johntwyman I believe you all might be interested in this

### CODE REVIEW

File: CRM/Core/DAO/Address.php [hunk 1]
```diff
@@ -767,6 +774,26 @@ public static function &fields() {
...
+          'dataPattern' => '',
+          'table_name' => 'civicrm_address',
+          'entity' => 'Address',
+          'bao' => 'CRM_Core_BAO_Address',
+          'localizable' => 0,
+          'html' => [
+            'type' => 'Select',
+          ],
+          'pseudoconstant' => [
+            'callback' => 'CRM_Core_SelectValues::getDisavilityAccessLevels',
```
Comment (eileenmcnaughton) [inline CRM/Core/DAO/Address.php]: dis-civility?

**Comment by eileenmcnaughton** [comment]
@seamuslee001 interesting idea - I think you should drop an email to the partner list. My first instinct on reading this was that it was more like 'has a scary dog' , 'leave parcels on back porch', - which makes me want to go against the grain & argue for an option group list.

**Comment by jaapjansma** [comment]
See my comment on the issue, I do think it is better to solve this with custom fields and/or in an extension.

**Comment by JoeMurray** [comment]
dev list also has feedback that this better in an extension for now. Closing until there is more consensus that this is ready for prime time in core.

**Pull request closed without merging**